### DISCUSSION TITLE
[1.19.1rc1]  Mixin transformation of net.minecraft.server.PlayerManager failed

### DISCUSSION THREAD

**Discussion by kwpugh** [FabricMC/fabric-api#2374] [Mod Dev Support] [answered]
Not sure if this is an  issue on my end, but I have tried to update two of my mods from 1.19 to latest 1.19.1rc1 and get the same crash in each:

Crash report: https://gist.github.com/kwpugh/5591a8e310938d17a8ebb27f98da5dd0

Using these values  from wiki;

minecraft_version=1.19.1-rc1
yarn_mappings=1.19.1-rc1+build.5
loader_version=0.14.8

#Fabric api
fabric_version=0.57.1+1.19.1

**Comment by warjort** [comment]
I get the same, I raised this bug report. https://github.com/FabricMC/fabric/issues/2376

You might be able to ignore it for now if you change your dependency to something like
````
 modImplementation("net.fabricmc.fabric-api:fabric-api:${project.fabric_version}") {
      exclude group: "net.fabricmc.fabric-api", module: "fabric-message-api-v1"
    }
````
assuming you don't use the message api. I haven't tried this.

**Reply by kwpugh** [reply]
Thanks for the heads up.  I will wait until it is fixed.

**Comment by modmuss50** [comment, marked as answer]
0.57.1+1.19.1 is designed for pre2 and not the **older** rc1. For pre1 and rc1 use the stable 1.19.0 version of fabric API, and for rc2 use the new 1.19.1 branch.

Mojang going backwards on version numbers mades this a bit of a mess/confusing.

**Comment by warjort** [comment]
Your response is just as confusing :-) 
I think you mean pre2 not rc2?

This seems to work for me
````
	minecraft_version=1.19.1-pre2
	yarn_mappings=1.19.1-pre2+build.2
	loader_version=0.14.8
	fabric_version=0.57.1+1.19.1
````

**Reply by kwpugh** [reply]
Yes, using 1.19.1-pre2 cleared it up, thank you.
